addon-test-support/extend/find-many.js

```diff
@@ -0,0 +1,36 @@
+import { getExecutionContext } from '../-private/execution_context';
+
+/**
+ * @public
+ *
+ * Returns array of elements
+ *
+ * @example
+ *
+ * import { findMany } from '../extend';
+ *
+ * export default function count(selector, options = {}) {
+ *   return {
+ *     isDescriptor: true,
+ *
+ *     get() {
+ *       return findMany(this, selector, options).length;
+ *     }
+ *   };
+ * }
+ *
+ * @param {Ceibo} pageObjectNode - Node of the tree
+ * @param {string} targetSelector - Specific CSS selector
+ * @param {Object} options - Additional options
+ * @param {boolean} options.resetScope - Do not use inherited scope
+ * @param {string} options.contains - Filter by using :contains('foo') pseudo-class
+ * @param {number} options.at - Filter by index using :eq(x) pseudo-class
+ * @param {boolean} options.last - Filter by using :last pseudo-class
+ * @param {boolean} options.visible - Filter by using :visible pseudo-class
+ * @param {string} options.testContainer - Context where to search elements in the DOM
+ * @return {Array} of Element
+ */
+export function findMany(pageObjectNode, targetSelector, options = {}) {
+  const opts = Object.assign({}, options, { multiple: true });
```

Comment:
I've just checked which options are comming here... hope I haven't missed any:

```js
  testContainer,
  resetScope,

  // needed for collections
  at,

  // needed for clickOnText
  contains,
  last,

  // not documented
  pageObjectKey,
  scope,

  // hm...
  visible
```

I think we should explicitly pick all the known `options` here and test each of them.

Reply:
👍  will add